community/world.py

```diff
@@ -231,6 +232,8 @@ def __init__(self, player, texture_manager, options):
 
 		self.pending_chunk_update_count = 0
 		self.chunk_update_counter = 0
+		self.c = 0
```

Comment:
What does `c` mean? Could this be a more explicit variable name?

Reply:
subchunk count, I agree it should be renamed
I called it C because thats how its called in the debug screen but I forgot to rename it

Reply:
Wait actually that isn't it
I have no idea

Reply:
Welp should I picked a better name from the start but I guess I did this at 2 AM

Reply:
Removed it since it servers no purpose